packages/tools/generator-widget/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+All notable changes to this widget will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+## Changed
+- Set required node version to v12 or higher.
```

Comment:
Internal question: is this a required for NPM >= v7?

Reply:
Node 12 was always required by PIW tools package. Moreover, Node 10 is EOL at the end of this month.

Reply:
So this is more like a fly-by in this PR to update the node dependency rather than being part of the npm v7 fixes?

Reply:
Correct